lib/graphql/transaction.ts

```diff
@@ -91,6 +98,19 @@ export class Transaction extends IterableModel<TransactionModel> {
     return new ResultPage(this, transactions, pageInfo, args);
   }
 
+  /**
+   * Fetches first 50 matching transactions for a given user input
+   * It will consider case insentsitive like matches for amount,
```

Comment:
nit: typo "insentsitive"

Reply:
Thank you 🙂 c8d7e7dec33255d94d7e49a1b6c3d54ac3a484f9